vendor/golang.org/x/crypto/ed25519/BUILD.bazel

```diff
@@ -2,9 +2,15 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
 go_library(
     name = "go_default_library",
-    srcs = ["ed25519.go"],
+    srcs = [
+        "ed25519.go",
+        "ed25519_go113.go",
+    ],
     importmap = "k8s.io/kops/vendor/golang.org/x/crypto/ed25519",
     importpath = "golang.org/x/crypto/ed25519",
     visibility = ["//visibility:public"],
-    deps = ["//vendor/golang.org/x/crypto/ed25519/internal/edwards25519:go_default_library"],
+    deps = [
+        "//vendor/crypto/ed25519:go_default_library",
```

Comment:
This is the line that is causing the error. Looking at the added comments in the file below this, I think it might be a go version issue. Are you able to rerun `make gomod` with go 1.12?

https://github.com/kubernetes/kops/blob/release-1.16/go.mod#L3

Reply:
yeah it is maybe the problem.. compiled now with 1.12.9